Fix failing Cypress frontend test CI run

<!--
  Before opening a new issue, please search existing issues:  https://github.com/reposense/RepoSense/issues
-->
### Please include the steps to reproduce the bug.
Check out Cypress frontend test section of a CI run on a open PR, for example:
https://github.com/reposense/RepoSense/actions/runs/13084422177/job/36513827724?pr=2270

### What was expected to happen?
Cypress frontend test should run successfully

### What actually happened? Please include a screenshot of the output.
```
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/m/mesa/libgbm-dev_24.0.9-0ubuntu0.3_amd64.deb  404  Not Found [IP: 40.81.13.82 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Fetched 12.3 MB in 5s (2353 kB/s)
Error: Process completed with exit code 100.
```

### If possible, include the URL to your RepoSense report or log files (if any).
Other PRs failing this test:
- https://github.com/reposense/RepoSense/actions/runs/13078822840/job/36513934795
- https://github.com/reposense/RepoSense/actions/runs/13084435419/job/36513643449?pr=2271

--- a/.github/workflows/integration.yml
+++ b/.github/workflows/integration.yml
@@ -156,7 +156,9 @@ jobs:
       run: chmod +x gradlew
 
     - name: Set up environment
-      run: sudo apt-get install libgtk2.0-0t64 libgtk-3-0t64 libgbm-dev libnotify-dev libnss3 libxss1 libasound2t64 libxtst6 xauth xvfb
+      run: |
+        sudo apt-get update
+        sudo apt-get install libgtk2.0-0t64 libgtk-3-0t64 libgbm-dev libnotify-dev libnss3 libxss1 libasound2t64 libxtst6 xauth xvfb
 
     - name: Run frontend tests
       run: ./gradlew frontendTest -Pci

